packages/classic/src/propNames.js

```diff
@@ -1,7 +1,10 @@
 'use strict'
 
+import { data } from '@skele/core'
+
```

Comment:
Hm, isn't `propNames` also a good candidate to go into core?

Reply:
Yes, I was thinking about that, and wanted to check up first. For sure `children` should be part of core, but I was not sure about some of the propNames that will be used by other subsystems.

For now, I moved the essential to core, but I left `action` and `read` related props in classic. I really didn't want to move the action one, since it had old-school cursor based logic.

Let me know if you don't agree and want me to change something.